gulpfile.js

```diff
@@ -0,0 +1,35 @@
+var gulp = require('gulp'),
+  rename = require('gulp-rename'),
+  react = require('gulp-react'),
+  path = require('path'),
+  browserSync = require('browser-sync'),
+  reload = browserSync.reload;
+  
+var paths = {
+  startUrl: '/lesson_1/lesson.html',
+  scripts: ['lesson_*/src/*.js']
+};
+ 
+gulp.task('server', function() {
+  browserSync({
```

Comment:
Out of curiosity, any reason to use  `browser-sync` instead of `gulp-connect` here?